app/models/candidate.rb

```diff
@@ -28,6 +32,18 @@ def prepare(user)
     end
   end
 
+  def editable?
+    self.election.view_status == 2 || post.elected_by == "Studierådet"
```

Comment:
Redundant `self` detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.